frontend/src/metabase-lib/queries/drills/underlying-records-drill.js

```diff
@@ -13,12 +13,14 @@ export function underlyingRecordsDrill({ question, clicked }) {
     return null;
   }
 
-  // the metric value should be the number of rows that will be displayed
-  const rowCount = typeof clicked.value === "number" ? clicked.value : 2;
+  // here we expect rows count of an aggregated query item, but actually we have numeric metric value, so
+  // we don't really know number of rows in original un-aggregated data
+  const value = typeof clicked.value === "number" ? clicked.value : 2;
```

Comment:
I'm not familiar with the notion of `clicked.value` much. So I'm still skeptical about the name change. Can you elaborate more what this value `clicked.value` really represents?